pyartcd/pyartcd/pipelines/brew_scan_osh.py

```diff
@@ -0,0 +1,75 @@
+"""
+[STORES STATE IN REDIS]
```

Comment:
I would remove this, since it can get stale in the future - we are better off reading code to see what happens where

Reply:
https://github.com/openshift-eng/art-tools/pull/51/commits/7c715b307aa02efa5d521ca0e931c9a2ae18766e